scripts/tizen/4.0/packaging/caffe2.spec

```diff
@@ -0,0 +1,180 @@
+Name:           caffe2
+Version:        0.7.0
+Release:        0
+Summary:        A deep learning, cross platform ML framework with the flexibility of Python and the speed of C++.
+Group:          Development/Tools
+License:        Apache-2.0
+URL:            https://github.com/caffe2/caffe2
+Distribution:   Tizen
+Vendor:         Samsung Electronics
+Packager:       Geunsik Lim <geunsik.lim@samsung.com>
+Source0:        %{name}-%{version}.tar.gz
+Source1001:     %{name}.manifest
+Patch0:         %{name}-eigen3-path.patch
+
+BuildRequires:  make cmake gcc binutils glibc glibc-devel cpp libstdc++
+BuildRequires:  protobuf-devel
+BuildRequires:  eigen3-devel
+BuildRequires:  openblas-devel
+BuildRequires:  lmdb-devel
+BuildRequires:  leveldb-devel
+BuildRequires:  gflags-devel
+BuildRequires:  boost-devel
+BuildRequires:  opencv-devel mesa
+
+BuildRequires:  python-python-gflags
+BuildRequires:  python = 2.7
+BuildRequires:  python-numpy-devel >= 1.7
+BuildRequires:  python-pybind11-devel
+
+Requires:       libleveldb
+Requires:       libopenblas0
+Requires:       gflags
+Requires:       glog
+Requires:       boost
```

Comment:
For shared-lib dependencies, you normally don't need to add "Requires". Actually, it is not recommended to add them.

Reply:
Thanks. Fixed.  Re-submitted with squash.

Reply:
I'm not seeing such a squashing commit.

Reply:
It's strange. I have run again that to remove the below lines. 
```bash
Requires:       libopenblas0
Requires:       gflags
Requires:       glog
Requires:       boost
```